components/ContactConfirmation.js

```diff
@@ -0,0 +1,19 @@
+import { renderRichText } from '../lib/richText';
+
+export default function ContactConfirmation({ contactConfirmationData }) {
+  const {
+    contact_confirmation_title_regular_1: contactConfirmationTitleRegular1,
+    contact_confirmation_title_italic_2: contactConfirmationTitleItalic2,
+    contact_confirmation_title_regular_3: contactConfirmationTitleRegular3,
+    contact_confirmation_description: description,
+  } = contactConfirmationData;
```

Comment:
I'm not sure if this is related to this PR, but there's a type bug with Safari, see below:

Usually the fonts have to be configured a bit differently with the next.js settings, let me know if you need help here

https://user-images.githubusercontent.com/29356116/113229123-4bbd1580-9264-11eb-8e94-ed74669ce005.mov

Reply:
I don't think it's necessarily related to this PR since we did styling in a separate PR but we will look into this and make it a task

Reply:
@josephmannis  Going to resolve this comment for now and pulling into a task